pkg/dynamic/field.go

```diff
@@ -29,11 +30,20 @@ func LookupSymbolField(rs reflect.Value) (string, bool) {
 
 // Used by bbgo/interact_modify.go
 func GetModifiableFields(val reflect.Value, callback func(tagName, name string)) {
+	if val.Kind() == reflect.Ptr {
+		val = val.Elem()
+	}
+	if !val.IsValid() {
+		return
+	}
 	for i := 0; i < val.Type().NumField(); i++ {
```

Comment:
cache this val.Type() in valueType variable?

Reply:
done